sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -17,18 +17,25 @@
 
 package org.apache.spark.sql.catalyst.util
 
-import java.sql.{Timestamp, Date}
+import java.sql.{Date, Timestamp}
 import java.text.SimpleDateFormat
 import java.util.{Calendar, TimeZone}
 
 import org.apache.spark.sql.catalyst.expressions.Cast
 
 /**
- * Helper function to convert between Int value of days since 1970-01-01 and java.sql.Date
+ * Helper function to convert between Int value of days since epoch and java.sql.Date,
```

Comment:
I think that we should re-word this slightly.  I'd say something like this (provided it's accurate):

> Helper functions for converting between internal and external date and time representations.
> 
> Dates are exposed externally as java.sql.Date and are represented internally as the number of dates since the Unix epoch (1970-01-01).  Timestamps are exposed externally as java.sql.Timestamp and are stored internally as longs, which are capable of storing timestamps with 100 nanosecond precision.